Source/WebKit/Shared/WebFindOptions.h

```diff
@@ -42,6 +42,7 @@ enum class FindOptions : uint16_t {
     NoIndexChange = 1 << 9,
     AtWordEnds = 1 << 10,
     DoNotSetSelection = 1 << 11,
+    IgnoreInertNodes = 1 << 12,
```

Comment:
Oh, you also need to teach `WebCore::FindOption core(OptionSet<FindOptions> options)` about this new option.